test/parallel/test-eventtarget-whatwg-customevent.js

```diff
@@ -0,0 +1,41 @@
+// Flags: --expose-internals
+
+'use strict';
+
+// eslint-disable-next-line no-unused-vars
+const common = require('../common');
+
+const {
+  Event,
+  EventTarget,
+} = require('internal/event_target');
+
+const { strictEqual, throws, equal } = require('assert');
+
+// Manually converted from https://github.com/web-platform-tests/wpt/blob/master/dom/events/CustomEvent.html
+// in order to define the `document` ourselves
+
+{
+  const type = 'foo';
+  const target = new EventTarget();
+
+  target.addEventListener(type, (evt) => {
```

Comment:
IIUC
```suggestion
  target.addEventListener(type, common.mustCall((evt) => {
```
(this also removes the need for `// eslint-disable-next-line no-unused-vars`)

Reply:
nice find! added